plugin/evm/vm.go

```diff
@@ -170,6 +170,77 @@ func init() {
 	log.Root().SetHandler(log.LvlFilterHandler(log.LvlDebug, log.StreamHandler(originalStderr, log.TerminalFormat(false))))
 }
 
+type exportTxFunc func(assetID ids.ID, // AssetID of the tokens to export
+	amount uint64, // Amount of tokens to export
+	chainID ids.ID, // Chain to send the UTXOs to
+	to ids.ShortID, // Address of chain recipient
+	baseFee *big.Int, // fee to use post-AP3
+	keys []*crypto.PrivateKeySECP256K1R, // Pay the fee and provide the tokens
+) (*Tx, error)
+
+type parseAddressFunc func(addrStr string) (ids.ID, ids.ShortID, error)
+
+type AtomicTransactor struct {
```

Comment:
`AtomicTransactor` implementation. It is based on the `export` call in the `service` object.